service/policy/db/migrations/20240805000001_drop_deprecated_resources_table.sql

```diff
@@ -0,0 +1,26 @@
+-- +goose Up
+-- +goose StatementBegin
+
+-- Remove the 'resources' table that was never used in platform 2.0 and should be removed
+
+DROP TABLE IF EXISTS resources;
+
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS resources
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    namespace VARCHAR NOT NULL,
+    version INTEGER NOT NULL,
+    fqn VARCHAR,
+    labels JSONB,
+    description VARCHAR,
+    policytype VARCHAR NOT NULL,
+    resource JSONB
+);
```

Comment:
This is interesting. Not sure what the default behavior is for a migration the removes deprecated stuff but I feel like this might be unnecessary. :shrug:

Reply:
I agree that it's unnecessary, but I added it to keep with the existing pattern so that "every up migration has a down migration". I don't think we've encountered this scenario yet. 🤷

Reply:
I think I'm going to leave it because, while it seems useless to add a down migration to add back an unused table, if someone else added a migration for something they believed was unused, I think I'd want them to add a down migration in case it was actually used unknowingly. It's probably best to just reinforce the practice. 🤔